dvc/scm/git/tree.py

```diff
@@ -30,7 +30,6 @@ def tree_root(self):
         return self.git.working_dir
 
     def open(self, path, binary=False):
-
```

Comment:
How are these whitelines related to the patch? Please avoid polluting your commits with unrelated changes.

Reply:
not related at all, my mistake.